README.md

```diff
@@ -1,26 +1,63 @@
-# streetscapes
-This repository contains information and code for retrieving and using data from the [global-streetscapes](https://github.com/ualsg/global-streetscapes/tree/main) dataset. 
+# `streetscapes`
+This repository contains information and code for retrieving and using data from the [global-streetscapes](https://github.com/ualsg/global-streetscapes/tree/main) dataset.
 
-### Converting global-streetscapes to parquet
+# Installation
+Clone the repository, enter the root directory and use pip to install the `streetscapes` package:
 
-`convert_csv_to_parquet.py` contains the code for reproducing `streetscapes-data.parquet` dataset. 
-For input you will have to download the original csv files from [hugging face](https://huggingface.co/datasets/NUS-UAL/global-streetscapes/tree/main/data). The code looks for these csv files in `../streetscapes-data`, which you may need to create yourself or change the path to the location of the files. 
-To limit size, the dataset contains the following data:
+```shell
+$> git clone git@github.com:Urban-M4/streetscapes.git
```

Comment:
Add that a virtual environment should be created.

Reply:
Done!